src/presentational-components/shared/top-toolbar.js

```diff
@@ -1,23 +1,23 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
-import { Level, LevelItem, Text, TextContent, TextVariants  } from '@patternfly/react-core';
-import clsx from 'clsx';
+import { Text, TextContent, TextVariants  } from '@patternfly/react-core';
 import { ToolbarTitlePlaceholder } from './loader-placeholders';
 import RbacBreadcrumbs from './breadcrubms';
+import { PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
```

Comment:
Can you use direct imports? So we can start trimming the bundle size. `import { PageHeader, PageHeaderTitle } from 
 '@redhat-cloud-services/frontend-components/components/PageHeader'`